docs/adapters/http.md

```diff
@@ -34,6 +38,10 @@ Parameter         |             Description          | Required?
 `-includeHeaders` | When set to true the headers from the response are appended to each of the points in the response | No; default: `false`
 `-timeField`      | Name of the field in the data which contains a valid timestamp  | No; defaults to `time`
 `-format`         | HTTP data format, supports: `csv`, `json` and `jsonl` | No; defaults to HTTP response header `Content-Type`
+`-followLinkHeader` |  When set to true the [Web Linking](https://tools.ietf.org/html/rfc5988) feature is enabled which allows for automatic handling of paging as specified by the [RFC5988](https://tools.ietf.org/html/rfc5988). | No; default `false`
+`-pageParam`      | The name of the query parameter used to enable paging with multiple HTTP requests sent to the URL provided until response contains 0 records | No
+`-pageStart`      | Initial value of the paging offset to set the `pageParam` to in the URL used to make the HTTP request | No; default: `0`
+`-pageUnit`       | Can be either `request` or `record`. If set to `request`, then each successive HTTP paging request increments the paging query parameter by one. If set to `record`, then each successive request increments the paging query parameter by the number of records returned in the previous request. | No; default: `request`
```

Comment:
we usually specify the value choices in the "Parameter" column, let's do like this:
```
`-pageUnit <request|record>`    | If set to `request`, then ...
```